Client/Frontend/Browser/Tab.swift

```diff
@@ -709,6 +709,14 @@ class TabWebView: WKWebView, MenuHelperInterface {
 
         return super.hitTest(point, with: event)
     }
+    
+    /// Override evaluateJavascript - should not be called directly on TabWebViews any longer
+    // We should only be calling evaluateJavascriptInDefaultContentWorld in the future
+    @available(*, unavailable, message:"Do not call evaluateJavaScript directly on TabWebViews, should only be called on super class")
```

Comment:
we don't want evaluateJavaScript called on the superclass, if we could block that too we would, simply saying to "use evaluateJavascriptInDefaultContentWorld() instead for greater safety" is better here